nav2_collision_monitor/src/polygon.cpp

```diff
@@ -286,6 +299,15 @@ bool Polygon::getCommonParameters(std::string & polygon_pub_topic)
       slowdown_ratio_ = node->get_parameter(polygon_name_ + ".slowdown_ratio").as_double();
     }
 
+    if (action_type_ == LIMIT) {
+      nav2_util::declare_parameter_if_not_declared(
+        node, polygon_name_ + ".linear_limit", rclcpp::ParameterValue(0.5));
+      linear_limit_ = node->get_parameter(polygon_name_ + ".linear_limit").as_double();
+      nav2_util::declare_parameter_if_not_declared(
+        node, polygon_name_ + ".angular_limit", rclcpp::ParameterValue(0.5));
```

Comment:
Is 0.5 a reasonable default for angular velocities? I think that seems a bit low

Reply:
On a circular robot, 0.5rad/s may be slow, but for a robot that needs to check collisions when turning it is a reasonable value when an obstacle is in the limited area.

For the linear part, the safety devices are often set with multiple polygons with decreasing velocity limits when closer to the robot. 0.5m/s is a middle choice between the possible values.